libsrc/target/aquarius/far/__far_page.asm

```diff
@@ -15,21 +15,19 @@ __far_page:
     and     a
     jr      z,localfar
     ld      d,e
-    dec     d
     ld      a,b
     rla
     rl      d
     rla
     rl      d
-    ; e is now which bank we should look at
+    ; d is now which bank we should look at
     ld      a,b
     and     @00111111      ;Take mod 16384
     or      @11000000	   ;Map to 0xc000 page
     ld      h,a
     ld      l,c
     ; hl = offset within bank
     ld      a,d
-    add     AQPLUS_FIRST_BANK	;0x01xxxx starts at AQPLUS_FIRST_BANK
```

Comment:
No need to +AQPLUS_FIRST_BANK as actual RAM page numbers are now used. Just need to shift the page number.